pandas_datareader/oandarest.py

```diff
@@ -25,24 +25,78 @@ class OANDARestHistoricalInstrumentReader(_BaseReader):
             Date to begin fetching curerncy pair, in RFC3339 ("%Y-%m-%dT%H:%M:%SZ)  # Eg: "2014-03-21T17:41:00Z"
         end: string
             Date to end fetching curerncy pair, in RFC3339 ("%Y-%m-%dT%H:%M:%SZ)  # Eg: "2014-03-21T17:41:00Z"
-        granularity: string
-            Time range for each candlestick
-            Default: See DEFAULT_GRANULARITY
+        freq: string
+            Frequency or periodicity of the candlesticks to be retrieved
+            Valid values are the following Panda's Offset Aliases (http://pandas.pydata.org/pandas-docs/stable/timeseries.html):
+                    5S  ->  5 second candlesticks, minute alignment
+                    10S -> 10 second candlesticks, minute alignment
+                    15S -> 15 second candlesticks, minute alignment
+                    30S -> 30 second candlesticks, minute alignment
+                    1T,1min  ->  1 minute candlesticks, minute alignment
+                    2T,2min  ->  2 minute candlesticks, hour alignment
+                    4T,4min  ->  4 minute candlesticks, hour alignment
+                    5T,5min  ->  5 minute candlesticks, hour alignment
+                    15T,15min -> 15 minute candlesticks, hour alignment
+                    30T,30min -> 30 minute candlesticks, hour alignment
+                    1H  ->  1 hour candlesticks, hour alignment
+                    2H  ->  1 hour candlesticks, day alignment
+                    3H  ->  3 hour candlesticks, day alignment
+                    4H  ->  4 hour candlesticks, day alignment
+                    6H  ->  6 hour candlesticks, day alignment
+                    8H  ->  8 hour candlesticks, day alignment
+                    12H -> 12 hour candlesticks, day alignment
+                    1D  ->  1 day candlesticks, day alignment
+                    1W  ->  1 week candlesticks, aligned to start of week
+                    1M  ->  1 month candlesticks, aligned to first day of the month
+                    See OANDA REST v20 for updated list
+            Default: See DEFAULT_FREQUENCY
         candleFormat: string
             Candlesticks representation
             Valid values: M,B,A
+                M for midpoint
+                B for Bid
+                A for Ask
             Default: See DEFAULT_CANDLE_FORMAT
         access_credential: Dict of strings
             Credential to query the api
             credential["accountType"]="practise". Mandatory. Valid values: practice, live
             credential["apiToken"]="Your OANDA API token". Mandatory. Valid value: See your OANDA Account's API Token
     """
-    DEFAULT_GRANULARITY="S5"
+    DEFAULT_FREQUENCY="5S"
     DEFAULT_CANDLE_FORMAT="M"
+    SUPPORTED_OFFSET_ALIASES = {
+            "5S":"S5",
+            "10S":"S10" ,
+            "15S":"S15",
+            "30S":"S30",
+            "1T":"M1",
+            "1min":"M1",
+            "2T":"M2",
+            "2min":"M2",
```

Comment:
I'm not sure a DateOffset can be convert back to a "normalized" offset string

```
In [20]: offset = to_offset("7T")

In [21]: offset.
                 offset.apply       offset.delta       offset.kwds        offset.nanos       offset.rollback
                 offset.apply_index offset.freqstr     offset.n           offset.normalize   offset.rollforward
                 offset.copy        offset.isAnchored  offset.name        offset.onOffset    offset.rule_code
```

but it could be an interesting feature to have in Pandas

```python
In [18]: offset.rule_code
Out[18]: 'T'

In [19]: offset.rule_code
Out[19]: 'T'

In [20]: offset = to_offset("7T")

In [21]: offset.n
Out[21]: 7

In [22]: offset.rule_code
Out[22]: 'T'

In [23]: offset = to_offset("7Min")

In [24]: offset.n
Out[24]: 7

In [25]: offset.rule_code
Out[25]: 'T'
```

a quick and dirty approach could be

```
In [47]: def normalized_offset_string(offset):
    ...:      if offset.n == 1:
    ...:          return offset.rule_code
    ...:      else:
    ...:          return str(offset.n) + offset.rule_code
```